ckanext/ontario_theme/plugin.py

```diff
@@ -1122,7 +1160,38 @@ def organization_facets(self, facets_dict, organization_type, package_type):
     def before_search(self, search_params):
         u'''Extensions will receive a dictionary with the query parameters,
         and should return a modified (or not) version of it.
+        Sets default access level to open
         '''
+        fl = search_params.setdefault("facet.field", [])
+        fq = search_params.get("fq", "")
+        default_open = '{!bool tag=orFqaccess_level should=\'access_level:"open"\'}'
+
+        ors = set(_get_default_ors())
+        # To show all access levels on dataset counts on
+        # the homepage and org/group pages
+        if ('fq' not in search_params) or (fq and not fl):
+            fq_list = search_params.setdefault('fq_list', [])
```

Comment:
`fq_list` is set here to `[]` but is never used again outside this `if` statement

Reply:
This is because the `fq_list` is used with `setdefault()`. The default value for fq_list is being set and modified within the if statement, so there is no need to use it outside the if statement, because it is being updated directly into `search_params`